plugins/frontend/backstage-plugin-jira/src/api/index.ts

```diff
@@ -217,4 +229,33 @@ export class JiraAPI {
       ),
     ];
   }
+
+  async getIssuesCountByJql(query: CustomQuery) {
+    const { apiUrl } = await this.getUrls();
+
+    // TODO : mutualize this ?
+    const request = await fetch(`${apiUrl}search`, {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json',
+      },
+      body: JSON.stringify({
+        jql: query.query,
+        maxResults: 0,
+      }),
+    });
+
+    if (!request.ok) {
+      throw new Error(
+        `failed to fetch data, sttatus : ${request.status}: ${request.statusText}`,
+      );
+    }
+
+    const response = await request.json();
+    return {
+      total: response.total,
+      name: query.title,
+      iconUrl: '', // TODO : find an icon
```

Comment:
In my implementation I used another component to render the status (CustomQueryStatus) which doesn't require an iconUrl as it uses one from material-ui.

If using this custom status component is kept, maybe the creation of a new type without the `iconUrl` property should be done. Another possibility could be to make this property optional in the already existing type.